src/jit/objectalloc.h

```diff
@@ -53,6 +53,7 @@ inline ObjectAllocator::ObjectAllocator(Compiler* comp)
     , m_IsObjectStackAllocationEnabled(false)
     , m_AnalysisDone(false)
 {
+    doChecks = false;
```

Comment:
Add a comment indicating why we don't do checks?

Reply:
Done.